VAMobile/src/store/api/demo/mocks/benjaminAdams/appointments.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
For these mocks where there's nothing happening, can we create a default that can be imported so we don't have to duplicate them for every user? ie default/appointments.json instead of having appointments for every user

Reply:
+1